impl/src/planning.rs

```diff
@@ -486,6 +491,13 @@ impl<'planner> CrateSubplanner<'planner> {
         }
       }
     }
+
+    // Take the hashmap and create a vector of strings that will be inserted into the template.
+    let mut aliased_dependencies = Vec::new();
+    for (name, alias) in &renamed_deps {
+      aliased_dependencies.push(format!("\"{}\": \"{}\",", name.clone(), alias.clone()));
```

Comment:
Are the clone() calls needed here, or could the variables be borrowed?  &name, &alias

Reply:
This logic no longer lives in this function and now doesn't involve looping through a HashMap. Resolving.